backend/src/main/java/com/jobwait/spring/utils/Utils.java

```diff
@@ -0,0 +1,19 @@
+package com.jobwait.spring.utils;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+public class Utils {
+    public final static ObjectMapper mapper = JsonMapper.builder()
+            .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true)
+            .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_VALUES, true)
+            .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS, true)
+            .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false) // otherwise you get "2024-08-09" as
+                                                                              // [2024, 08, 09]
```

Comment:
this makes no sense to me. why would jackson considered dashes as delimiters for an array of elements?? Good catch

Reply:
it doesnt, this is specifically for timestamps like "2024-08-19" it would convert it to an array of elements, as to why, you would have to look at the specific library https://github.com/FasterXML/jackson-modules-java8/tree/2.14/datetime#:~:text=LocalDate%2C%20LocalTime%2C%20LocalDateTime%2C%20and%20OffsetTime%2C%20which%20cannot%20portably%20be%20converted%20to%20timestamps%20and%20are%20instead%20represented%20as%20arrays%20when%20WRITE_DATES_AS_TIMESTAMPS%20is%20enabled. as to why this is enabled by default in Jackson, im not sure